apps/src/craft/code-connection/blocks.js

```diff
@@ -604,16 +610,16 @@ export const install = (blockly, blockInstallOptions) => {
   };
 
   blockly.JavaScript.craft_fill = function () {
-    var fromPositionType = this.getTitleValue('FROMPOSITIONTYPE');
-    var fromX = this.getTitleValue('FROM_X');
-    var fromY = this.getTitleValue('FROM_Y');
-    var fromZ = this.getTitleValue('FROM_Z');
-    var toPositionType = this.getTitleValue('TOPOSITIONTYPE');
-    var toX = this.getTitleValue('TO_X');
-    var toY = this.getTitleValue('TO_Y');
-    var toZ = this.getTitleValue('TO_Z');
+    var fromPositionType = "\"" + this.getTitleValue('FROMPOSITIONTYPE') + "\"";
```

Comment:
Let's move the quotes down to line 622 so we don't need to escape this value.  E.g.:

```js
var fromPositionType = this.getTitleValue('FROMPOSITIONTYPE');
// ...
return `fill('block_id_${this.id}',createBlockPos(${fromX}, ${fromY}, ${fromZ}, "${fromPositionType}"), ...`
```